packages/plugin-resizer/src/components/ResizerComponent.ts

```diff
@@ -0,0 +1,89 @@
+import { OwlComponent } from '../../../plugin-owl/src/OwlComponent';
+import { Layout } from '../../../plugin-layout/src/Layout';
+import { DomLayoutEngine } from '../../../plugin-dom-layout/src/DomLayoutEngine';
+import { ZoneNode } from '../../../plugin-layout/src/ZoneNode';
+
+interface ResizerProps {
+    minHeight?: number;
+    height?: number;
+}
+
+export class ResizerComponent<T extends ResizerProps = {}> extends OwlComponent<T> {
+    static template = 'resizer';
+    domEngine = this.env.editor.plugins.get(Layout).engines.dom as DomLayoutEngine;
+    targetBodyEl: HTMLElement;
+
+    async mounted(): Promise<void> {
+        await super.mounted();
+        // Wait a tick to be able to find the main zone in the DOM.
+        // If we do not wait, the main zone is not yet present in the DOM.
+        await new Promise(resolve => {
+            setTimeout(resolve);
+        });
+
+        const mainZone = this.domEngine.root.descendants(
+            node => node instanceof ZoneNode && node.managedZones.includes('main'),
+        )[0];
+        const jwBody = (this.domEngine.getDomNodes(mainZone)[0] as HTMLElement)?.parentElement;
+        if (jwBody) {
+            this.targetBodyEl = jwBody as HTMLElement;
+            this.targetBodyEl.style.overflow = 'auto';
+        }
+        if (this.props.height) this._setJwBodyHeight(this.props.height);
```

Comment:
The Node should have absolutely no clue about the DOM. Especially a DOM Node outside it's direct.
There is a hidden and strong dependency with theses lines.